Library/Homebrew/cmd/update-bash.sh

```diff
@@ -303,6 +303,9 @@ EOS
       odie "Git must be installed and in your PATH!"
     fi
   fi
+  export GIT_TERMINAL_PROMPT="0"
+  export GIT_ASKPASS="false"
+  export GIT_SSH="$HOMEBREW_LIBRARY/ENV/scm/ssh"
```

Comment:
`export GIT_SSH_COMMAND="ssh -oBatchMode=yes"` so you don't need to create ssh script in superenv, which may interfere brew install or other unexpected places.